src/CplexSolverInterface.jl

```diff
@@ -615,10 +616,10 @@ function setmathproglazycallback!(model::CplexMathProgModel)
     set_param!(model.inner.env, "CPX_PARAM_MIPCBREDLP", 0)
     set_param!(model.inner.env, "CPX_PARAM_PRELINEAR", 0)
     set_param!(model.inner.env, "CPX_PARAM_REDUCE", CPX_PREREDUCE_PRIMALONLY)
-    cpxcallback = cfunction(mastercallback, Cint, (Ptr{Void}, Ptr{Void}, Cint, Ptr{Void}, Ptr{Cint}))
+    cpxcallback = cfunction(mastercallback, Cint, (Ptr{Nothing}, Ptr{Nothing}, Cint, Ptr{Nothing}, Ptr{Cint}))
```

Comment:
@cfunction

Reply:
It's still `cfunction` instead of `@cfunction`...

Reply:
`cfunction` does not exist in Julia 1.0

Reply:
That's why you should change it ...
The code still contains `cfunction`

Reply:
That was exactly the point of my comment :)

Reply:
Btw I still cannot use `@lazyconstraint`, it gives always the same error:
`MethodError: no method matching cbgetstate(::CPLEX.CplexLazyCallbackData)`

Reply:
Sorry Miles, I thought you were making the changes, I didn't understand was only a comment. -.-"
It's sunday, I'm working, please apologize.